TuneUp/TuneUpWindow.xaml.cs

```diff
@@ -41,10 +41,10 @@ public TuneUpWindow(ViewLoadedParams vlp, string id)
         {
             InitializeComponent();
             viewLoadedParams = vlp;
-            vlp.DynamoWindow.SizeChanged += DynamoWindow_SizeChanged;
             // Initialize the height of the datagrid in order to make sure
             // vertical scrollbar can be displayed correctly.
             this.NodeAnalysisTable.Height = vlp.DynamoWindow.Height - sidebarHeightOffset;
+            vlp.DynamoWindow.SizeChanged += DynamoWindow_SizeChanged;
```

Comment:
@QilongTang hmm, I guess this works quite well... unless we want to remove the DynamoWindow from the API - in that case it might make more sense to wrap this event in our own event... what do you think?

Reply:
Yes, we can definitely do that in Dynamo 3.0